snapshot/snapshot.go

```diff
@@ -122,7 +122,7 @@ func NewSnapshotter(ctx context.Context, cfg *config.SnapshotterConfig) (snapsho
 		}()
 	}
 
-	if cfg.EnableSystemController {
+	if config.SystemControllerEnabled() {
```

Comment:
Maybe `IsSystemControllerEnabled` or `GetSystemControllerEnabled` is better to understand?

Reply:
Renamed